web/modules/custom/portland/portland.module

```diff
@@ -31,7 +31,9 @@ const GROUP_CONTENT_TYPE_PLURAL = [
     "page" => ["pages"],
     "development_permit" => ["permits"],
     "location" => ["locations"],
-    "calendar_item" => ["calendar/archive"]
+    "calendar_item" => ["calendar/archive"],
+    "city_code" => ["code"],
```

Comment:
@jookyg, this should be removed as we don't need pluralization of code and charter for groups.